public/src/shared/_landing.scss

```diff
@@ -0,0 +1,58 @@
+#landing {
```

Comment:
also I would suggest seeding most of your scss for a component inside of the root div something like below. it helps make sure you don't clash with other class names

```
#store-tour {
    padding: 50px;
    display: flex;
    align-items: center;
    justify-content: center;


    .ratio-width {
        max-width: 1050px;
        width: 100%;
    }
}
```